.git-blame-ignore-revs

```diff
@@ -1,2 +1,8 @@
 # scalafmt
 90f8bf7e6d6f7a2d6c63313944613a95ff60091a
+
+# javafmt
```

Comment:
revert these file changes - the Pekko contributors will decide what goes in here